subr/macports.sh

```diff
@@ -142,9 +145,10 @@ make_package()
 	    version="$2"
 	    ;;
     esac
-    known_macos_db | awk -F'-' "-vmacos=${macos}" "-vversion=${version}" '
+    . "${subrdir}/check_version.sh" "${version}"
```

Comment:
I would rather approach this by moving the logic provided by check_version.sh to this file, be sure to define functions printing the desired values instead of storing the values in variables. Continue to use macports.sh everywhere you need your functions.